sdk/monitor/monitor-ingestion/sample.env

```diff
@@ -0,0 +1,11 @@
+# Retrieve the following values from your app registration in the Azure Portal.
+
+AZURE_TENANT_ID=""
+AZURE_CLIENT_ID=""
+AZURE_CLIENT_SECRET=""
+
+# Retrieve the following values from azure monitor resource in the Azure Portal.
+
+LOGS_INGESTION_ENDPOINT=""
+DATA_COLLECTION_RULE=""
+IMMUTABLE_ID=""
```

Comment:
Is this needed in the tests? I didn't see it in the karma config file.

Reply:
cleaned it up